src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/summary/SummaryGroupComponent.tsx

```diff
@@ -244,6 +244,7 @@ function SummaryGroupComponent(props: ISummaryGroupComponent) {
           return (
             // eslint-disable-next-line react/jsx-key
```

Comment:
Could you please remove this comment as well, as its not valid anymore (and really should not have been added in the first place either 😅)

Reply:
```suggestion
```